### ISSUE TITLE
Missing "cancel purchase" button

### ISSUE THREAD

**Issue by pavel4win** [Regalis11/Barotrauma#13189]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Previously, when purchasing goods, there was an additional question to buy or cancel the purchase. There was a “buy” button at the top and a “cancel” button at the bottom. Now there is no "cancel" option. Please add it. And swap the two “receive immediately”/“deliver to boat” buttons so that there is a “deliver to boat” button on top (as was in previous version). If there is not enough space in the inventory to "get goods immediately", “receive immediately” button should be disabled so your purchases do not fall out on the floor. When purchasing text screens and terminals for which there was not enough space in my inventory, I discovered that they were immediately attached to the wall of the station.

### Reproduction steps

1. Fill inventory with supplies
2. Go to merchant and buy many text screens
3. You have no choice to cancel the buy
4. Your will discover that the screens have been attached to the wall after purchase (if you not had free space in your inventory)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.6.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! There has never been a cancel button in the prompt, so this is more of a feature request than a bug report.

https://github.com/Regalis11/Barotrauma/assets/14351070/f848b4fc-1cc3-46d3-9ddf-1488e3fbef59

Items spawning attached to a wall is a bug though, I'll look into that!

**Comment by Regalis11** [comment]
Fixed the spawning issue in https://github.com/Regalis11/Barotrauma-development/commit/4e3c816f84c812bc9d1f2fb6e18d51fc5e60e5dc

**Comment by pavel4win** [comment]
You are right. I realized that confirmation occurs when items are sold, not when purchased. And since YES was located in the first line, here too I click first choice (as if it is YES) by mistake even when I want to "deliver items to the submarine". Now I'm more attentive.
